vagrant/provisioning/roles/arkcase-app/tasks/main.yml

```diff
@@ -116,10 +116,14 @@
   register: arkcase_war
   when: arkcase_retrieve_strategy == "sftp"
 
+- name: format config server version
+  set_fact:
+    arkcase_config_server_version_formatted: "{{ '-' ~ arkcase_config_server_version if arkcase_config_server_version != '' else '' }}"
```

Comment:
I knew the dash would be an issue, but I didn't like the versioning without the dash :)